pyiron_atomistics/atomistics/job/atomistic.py

```diff
@@ -413,6 +415,21 @@ def restart(self, job_name=None, job_type=None):
         new_ham._generic_input['structure'] = 'atoms'
         return new_ham
 
+    def check_if_file_exists(self, filename) -> None:
+        """
+        Checks if a given file exists within the job directory
+
+        ToDo: Move this to pyiron_base since this is more generic.
+
+        Args:
+            filename (str): The name of the file
+
+        Raises:
+            FileNotFoundError: Raised if the given file does not exist.
+        """
+        if not os.path.isfile(posixpath.join(self.working_directory, filename)):
+            raise FileNotFoundError(f"File {filename} not found")
```

Comment:
One should also mention the directory where the file should be. Otherwise, it might be confusing. Cf. my changes in #421

Reply:
Ok, but the working directory might only be created at `run()` time and a 'you did not copy a file to some unknown location' would be unfair. Thus, I would rather include the information on how to tell pyiron a file is required and should be copied to the working directory.

Reply:
@niklassiemer : this is an auxiliary function for preparing restart jobs from an existing job, and producing a comprehensible error if the files needed for a successful restart do not exist.
In this very situation, the working directory for the source job of the restart must exist (otherwise we cannot extract files from it).